adapters/lora-vertx-base/src/main/java/org/eclipse/hono/adapter/lora/providers/JsonBasedLoraProvider.java

```diff
@@ -97,7 +98,7 @@ public LoraCommand getCommand(final CommandEndpoint commandEndpoint, final Strin
      *
      * @return The JSON payload.
      */
-    protected JsonObject getCommandPayload(final Buffer payload, final String deviceId) {
+    protected JsonObject getCommandPayload(final Buffer payload, final String deviceId, final String subject) {
```

Comment:
please add JavaDoc tags for the parameters and the returned object

Reply:
added them

Reply:
added them